### DISCUSSION TITLE
Facing issue with "@itwin/itwinui-css": "^1.11.1" and "@itwin/map-layers": "^4.0.0-dev.44",  packages..

### DISCUSSION THREAD

**Discussion by Praful-Bentley** [iTwin/iTwinUI#1331] [Q&A]
Facing issue with "@itwin/itwinui-css": "^1.11.1" and "@itwin/map-layers": "^4.0.0-dev.44",  packages..

The itwinui-css package seems to be missing scss folder and corresponding files required in map-layers package.

Error: import file cannot be resolved: "@import "~@itwin/itwinui-css/scss/style/typography"

Can you please suggest?

![image](https://github.com/iTwin/iTwinUI/assets/135814108/b6cc1dd5-345a-47df-8ffc-bacfda150dc1)

**Comment by veekeys** [comment]
Answered here
https://github.com/iTwin/iTwinUI/issues/1332
